docs/user.md

```diff
@@ -489,7 +499,10 @@ spec:
     caFile: "ca.crt" # add this if the secret is configured with a CA
 ```
 
-Before applying these changes, the operator must also be configured with the
-`spilo_fsgroup` set to the GID matching the postgres user group. If the value
-is not provided, the cluster will default to `103` which is the GID from the
-default spilo image.
+Certificate rotation is handled in the spilo image which checks every 5
+minutes if the certificates have changed and reloads postgres accordingly.
+
+Known issues:
+
+* Existing clusters don't handle changes to the `spilo_fsgroup` well.
```

Comment:
That's not very precise. Are changes ignored? Does it brake the cluster?

Reply:
It inherits the same issue that changing `spilo_fsgroup` has where the pod needs to be re-created.

I will remove that section for now.

Reply:
Once the user reads-up on `spilo_fsgroup` they will probably find the issue.